src/widgets/current-refined-values/current-refined-values.js

```diff
@@ -272,12 +272,15 @@ export default function currentRefinedValues({
     const makeCurrentRefinedValues = connectCurrentRefinedValues(
       specializedRenderer
     );
-    return makeCurrentRefinedValues({
-      attributes,
-      onlyListedAttributes,
-      clearAll,
-      clearsQuery,
-    });
+    return Object.assign(
```

Comment:
Why not pass containerNode to make...* and then return it back as it was? Would be easier to understand I feel.

Reply:
`makeXXXX` is the connector which returns the widget instance. Since we decided to remove all the view logic from the connector it would be weird as a connector to have an entry for saving the container DOM element of the widget.

I have no real strong opinion on this one, it will probably change 👍